contributing_to_docs/doc_guidelines.adoc

```diff
@@ -239,10 +239,50 @@ just use the major version number. Do not prepend with a `v`, as in `v3` or `v4`
 
 Do not use markup in headings.
 
-*Do not use internal company server names in command or example output*. See
-suggested host name examples
-https://docs.openshift.com/container-platform/3.11/install/example_inventories.html#multi-masters-single-etcd-using-native-ha[here].
+== Host name formatting
```

Comment:
Although the node name might typically be the host name, it might not be immediately obvious to me (or maybe new people) that this applies to the node name. And maybe potentially other areas of the objects.

I'd suggest to give some examples of where this could be used, like the `oc get nodes` output we've been talking about.

Reply:
Thank you for highlighting this. I have updated most of the "host name" references to "node name", except where referring to provider host names.

I have also added an example node name output to demonstrate usage.